### PR TITLE
Process 2017: Rescinded and Obsolete RECs

### LINKED ISSUE #483: Obsoleting recommendations
There is a proposal to add the abiility to obsolete a Recommendation (this comes in addition to the current possibility of rescinding Recommendations):
https://dvcs.w3.org/hg/AB/raw-file/default/cover.html#rec-rescind

If Process 2016 gets adopted (current timeline is not established yet but no earlier than October 2016 at least), this will probably be a variant of the current rescinded profile.

### PR DESCRIPTION
**Pull request by deniak**
@tripu I'll add some tests soon but you can already review the PR

Fix #483

### CODE REVIEW

**Comment by deniak** [comment]
@tripu that PR should be ready to be reviewed. I'll submit another one for the final changes wrt process 2017.

cc @plehegar

File: lib/l10n-en_GB.js [hunk 1]
```diff
@@ -141,6 +147,9 @@ exports.messages = {
 ,   "sotd.ac-review.not-found":   "Did not find information about a forum for AC Representative feedback."
     // sotd/diff
 ,   "sotd.diff.note":   false
+    // sotd/rescind
+,   "sotd.obsl-rescind.no-rational":           "No rational section found."
```
Comment (tripu) [inline lib/l10n-en_GB.js]: `s/rational/rationale/gi` (appears in other files, too).

File: lib/rules/sotd/obsl-rescind.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// SotD
+// <p>W3C has chosen to (rescind|obsolete) the Sample Specification Recommendation for the
+// following reasons: [...list of reasons...]. For additional information about
+// replacement or alternative technologies, please refer to the <a
+// href="https://www.w3.org/2016/11/obsoleting-rescinding/">explanation of
+// Obsoleting and Rescinding W3C Specifications</a>.</p>
```
Comment (tripu) [inline lib/rules/sotd/obsl-rescind.js]: Just curious: where does this copy come from? I can't find it on [`w3.org/2015/Process-20150901`](https://www.w3.org/2015/Process-20150901/#rec-rescind) nor on [`w3c.github.io/w3process-obsolete-rescinded`](https://w3c.github.io/w3process-obsolete-rescinded/).

File: lib/l10n-en_GB.js (hunk 1 above)
Comment (deniak) [inline lib/l10n-en_GB.js]: Oups, fixed!

File: lib/rules/sotd/obsl-rescind.js (hunk 2 above)
Comment (deniak) [inline lib/rules/sotd/obsl-rescind.js]: Ah sorry, I forgot to mention that. That text was suggested by @koalie while I was preparing the mockups for [obsolete recs](https://www.w3.org/2017/02/tr-rscnd-obsl/obsoleted.html) and [rescinded recs](https://www.w3.org/2017/02/tr-rscnd-obsl/rescinded.html).

**Comment by deniak** [comment]
@tripu yes, that PR will close #483

**Comment by tripu** [comment]
Thanks, @deniak. All good now.

(My only doubt is: how will editors know what exact copy Specberus expects to find, when they want to rescind a spec, or mark a spec as obsolete? Are they supposed to contact the webmaster in person in such rare occasions, and will the webmaster then point them to those two mock-ups you mentioned? Or will these two profiles have their own section and rules in [`w3.org/pubrules/doc`](https://www.w3.org/pubrules/doc), at least to document these new chunks of text?)

Apart from that, OK to merge. (Tue 28 Feb, at the end of the day?)

**Comment by deniak** [comment]
Thanks for the review! The editors should indeed check the rules descriptions at [`w3.org/pubrules/doc`](https://www.w3.org/pubrules/doc). The new profile and and updated one will be listed there.

**Pull request merged**